### ISSUE TITLE
ReferenceError/ - adKill/adkilldelay is not defined

### ISSUE THREAD

**Issue by Plant-for-the-Planet** [Plant-for-the-Planet-org/treecounter-app#2387]
## Error in Plant-for-the-Planet App

```
ReferenceError adKill is not defined 
ReferenceError adkilldelay is not defined
```
[View on Bugsnag](https://app.bugsnag.com/plant-for-the-planet/plant-for-the-planet-app/errors/5edaae3807fac600179516c4?event_id=5edaae38005b6cdb7a310000&i=gh&m=ci)

## Stacktrace

    global code:1 - 

[View full stacktrace](https://app.bugsnag.com/plant-for-the-planet/plant-for-the-planet-app/errors/5edaae3807fac600179516c4?event_id=5edaae38005b6cdb7a310000&i=gh&m=ci)

*Created automatically via Bugsnag*

**Comment by norbertschuler** [comment]
`getBoundingClientRect` is not a function in our codebase.

**Comment by Plant-for-the-Planet** [comment]
An error linked to this issue has been reopened in Bugsnag
[**ReferenceError** in **/donate-trees/1**](https://app.bugsnag.com/plant-for-the-planet/plant-for-the-planet-app/errors/5edaae3807fac600179516c4?event_id=5efa3818005b79c57c5c0000&i=gh&m=ra)

**Comment by norbertschuler** [comment]
Not sure if this new error rising today is related to https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/2693 as I found these breadcrumbs:
<img width="858" alt="Bildschirmfoto 2020-10-21 um 17 35 53" src="https://user-images.githubusercontent.com/1532418/96743172-e714a980-13ed-11eb-9b9c-39626e0e9121.png">
(The fix for #2693 - a browser check and a polyfill for Object.values - is not yet released on production.)
